Automattic/wp-calypso
# Stats: Swapping Videos component

#### Proposed Changes

* using a feature flag and replacing `Videos` StatsModule with the recently created component with horizontal bars
* consolidating a component introduced while swapping `Posts & pages` #69545 and renaming it to `StatsListCard`

#### Testing Instructions



* Navigate to the main `Stats` page either on a local instance or the test URL from the comment below
* append `?flags=stats/new-stats-module-component` to the URL
* verify that `Video` component has horizontal bars

Before
![Screenshot 2022-11-04 at 1 56 08 PM](https://user-images.githubusercontent.com/112354940/199862418-3e177204-faa9-47a8-a3a7-b2bf06cd3c46.png)

After
![Screenshot 2022-11-04 at 1 55 09 PM](https://user-images.githubusercontent.com/112354940/199862436-344cf01c-5c58-4f24-b2e0-45c09257849f.png)


#### Pre-merge Checklist






Related to #69044
